.github/renovate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "github>openshift/boilerplate//.github/renovate.json"
```

Comment:
[Link to Boilerplate's file.](https://github.com/openshift/boilerplate/blob/69aeedc6c9b74626dc9531331dd4fe192ca13b9a/.github/renovate.json)

From what I can gather this means PRs to `pipelines/` and `.tekton/` would be automatically approved and merged. Is that for any user or is it limited to a specific service account?